tests/test_ui_manager.py

```diff
@@ -3,6 +3,7 @@
 import pygame
 import pytest
 
+from pygame_gui.elements import UIDropDownMenu, UIHorizontalScrollBar
```

Comment:
Are these new imports being used?

Reply:
Ah, I had initially put the new test here because I had some trouble navigating the project tree, I'll remove these imports.